src/utils/accessToken.ts

```diff
@@ -91,5 +94,22 @@ export const accessToken = {
     }
 
     return userId;
+  },
+
+  /**
+   * Asserts that the provided access token is a delegated access token.  
+   * @throws {CommandError} Will throw an error if the access token is not available.
+   * @throws {CommandError} Will throw an error if the access token is an application-only access token.
+   */
+  assertDelegatedAccessToken(): void {
+    const accessToken = auth?.connection?.accessTokens?.[auth.defaultResource];
+    if (!accessToken) {
+      throw new CommandError('No access token could be found.');
+    }
+
+    ;
+    if (this.isAppOnlyAccessToken(accessToken.accessToken)) {
```

Comment:
`accessToken.accessToken` looks a bit odd. Let's just append it to the `accessToken` variable on line 105.